src/networks/types.ts

```diff
@@ -3,7 +3,7 @@ import type { Signer } from '@ethersproject/abstract-signer';
 import type { MetaTransaction } from '@snapshot-labs/sx/dist/utils/encoding';
 import type { Space, SpaceMetadata, Proposal, Vote, User, Choice, NetworkID } from '@/types';
 
-export type PaginationOpts = { limit: number; skip?: number };
+export type PaginationOpts = { limit: number; skip?: number; filter?: any };
```

Comment:
Not really belongs in `PaginationOpts`, for proposal search I just added 3rd optional parameter for search (in that case it's just string for single property instead of dynamic object though).